### DISCUSSION TITLE
Minor: Should the Tailwind template be called Tailwind 3?

### DISCUSSION THREAD

**Discussion by darylknight** [solspace/craft-freeform#418] [Feature Requests]
I tried digging through the source code to verify this, but I can't tell what version you're actually using. You've got Bootstrap 3, 4 and 5 in there, and I noticed the Tailwind template has been recently named "Tailwind 1". Unless you are actually using that, Tailwind is currently on version 3. The differences aren't breaking, but it might be worth updating it just so it looks more recent :)

Craft 3.4.2.1
Freeform 4.0.0-beta.15

**Comment by kjmartens** [comment]
Thanks for your feedback @darylknight. It's actually on our list to include a `v3` of the Tailwind formatting template. Should be available soon. 🙂

**Reply by darylknight** [reply]
Sounds great, thanks!

**Reply by kjmartens** [reply]
This has now been added for Freeform 4.0.1+ 🙂

**Reply by darylknight** [reply]
Awesome, thanks!
